charts/astronomer/values.yaml

```diff
@@ -171,6 +171,36 @@ houston:
     # Only run on deployments marked as canary
     canary: false
 
+  # Cleanup airflow db data
+  # This runs as a CronJob
+  cleanupAirflowDb:
+    # Enable cleanup CronJob
+    enabled: false
+
+    # Default run is at 5:23 every morning https://crontab.guru/#23_5_*_*_*
+    schedule: "23 5 * * *"
+
+    # Cleanup deployments older than this many days
+    olderThan: 365
+
+    # Output path of archived data csv export
+    outputPath: "/tmp"
+
+    # Delete archived tables
+    purgeArchive: true
+
+    # Print out the deployments that should be cleaned up and skip actual cleanup
+    dryRun: false
+
+    # Name of file storage provider, supported providers - aws/azure/gcp/local
+    provider: local
+
+    # Name of the provider bucket name / local file path
+    bucketName: "/tmp"
+
+    # Connection ID for provider bucket
```

Comment:
Can we get some additional information here? Maybe a short explanation of what a Connection ID is, or a hyperlink to a more complete explanation?